src/hotspot/os/linux/os_linux.cpp

```diff
@@ -271,6 +271,16 @@ julong os::Linux::available_memory() {
   return avail_mem;
 }
 
+julong os::used_memory() {
+  if (OSContainer::is_containerized()) {
```

Comment:
I'm curious. What's the intended use case of this? https://bugs.openjdk.org/browse/JDK-8329758 perhaps? If this is getting used more broadly, https://bugs.openjdk.org/browse/JDK-8261242, would seem to get bumped in priority as that still affects physical Linux systems.

Reply:
I think you hit the nail on the head regarding intended use case. It's probably important that that issue gets resolved, yes.